pkg/kyverno/test/test_command.go

```diff
@@ -324,8 +324,18 @@ func testCommandExecute(dirPath []string, valuesFile string, fileName string, gi
 		}
 
 	} else {
+		var count int
+		var dirTestYamlNameCount int
```

Comment:
Since YAML is an acronym, please use uppercase to satisfy linters i.e. `dirTestYAMLNameCount`. 

However, it may be best to shorten the variable name to `testFileCount` or similar?

Reply:
Oh, I see now that this variable is used only to count the deprecated "test.yaml" files. So, perhaps "deprecatedFileCount"?